.idea/runConfigurations/_template__of_py_test.xml

```diff
@@ -45,4 +45,4 @@
     <option name="_new_targetType" value="&quot;PATH&quot;" />
     <method v="2" />
   </configuration>
-</component>
```

Comment:
There is an extra line at the end of this file. Might be easiest to rebase the fork and then push --force or something. Or you could simply copy and paste the contents of the file located in the main branch of ivy.